components/eamxx/src/physics/shoc/tests/shoc_brunt_length_tests.cpp

```diff
@@ -124,11 +121,11 @@ struct UnitWrap::UnitTest<D>::TestCompBruntShocLength {
     }
   }
 
-  static void run_bfb()
```

Comment:
What was the reason for making the run methods non-static? Not that I have a preference in case both are fine, I don't see any pros/cons. Just curious.